### ISSUE TITLE
Pootle Install 

### ISSUE THREAD

**Issue by jauzzz** [translate/pootle#6750]
I am a Pootle novice, I successfully installed it on the mac, but when i want to deploy to the production environment, the following installation error occurs.

# Steps to reproduce:

1. I create a virtualenv env
2. Install Pootle requirements
3. Install Pootle, using `pip install --process-dependency-links Pootle`

# Results (Expected/Actual):

```Python
Collecting Pootle
  Using cached Pootle-2.8.1.tar.bz2
    Complete output from command python setup.py egg_info:
    Pootle requires a non-PyPI dependency, when using pip ensure you use the --process-dependency-links option.
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-_2TvXs/Pootle/setup.py", line 453, in <module>
        long_description=parse_long_description(README_FILE, tag=True),
      File "/tmp/pip-build-_2TvXs/Pootle/setup.py", line 404, in parse_long_description
        replace_urls()
      File "/tmp/pip-build-_2TvXs/Pootle/setup.py", line 332, in replace_urls
        branch = version.get_git_branch()
      File "/tmp/pip-build-_2TvXs/Pootle/pootle/core/utils/version.py", line 231, in get_git_branch
        'HEAD']).strip()
    AttributeError: 'NoneType' object has no attribute 'strip'
```

# Environment (i.e. 'pootle --version', DB, OS, Browser):

python version: 2.7.12
system: ubuntu 16.04

**Comment by phlax** [comment]
hi @jauzzz the workaround for this is to install `git` into your system environment. 

I will land a fix shortly that makes this not necessary

**Comment by unho** [comment]
@jauzzz We have just released Pootle 2.8.2 with a fix for this bug. Thanks for reporting and our apologies for the inconveniences.

**Comment by jauzzz** [comment]
Sorry, just to see the reply, do not know what should be used to express my thanks, this is my first mention of the issue has been so enthusiastic help, really grateful
